src/commands/kick.ts

```diff
@@ -0,0 +1,53 @@
+import { Embed } from "./../utils/Embed.ts";
+import { Member } from "https://raw.githubusercontent.com/Skillz4Killz/Discordeno/v7/src/structures/member.ts";
+import { botCache } from "../../mod.ts";
+import { kick } from "https://raw.githubusercontent.com/Skillz4Killz/Discordeno/v7/src/handlers/member.ts";
+import { sendEmbed } from "../utils/helpers.ts";
+import { sendMessage } from "https://raw.githubusercontent.com/Skillz4Killz/Discordeno/v7/src/handlers/channel.ts";
+
+botCache.commands.set(`kick`, {
+  name: `kick`,
+  guildOnly: true,
+  arguments: [
+    {
+      name: "member",
+      type: "member",
+      required: true,
+      missing: (message) => {
+        return sendMessage(message.channel, "User not found!");
+      },
+    },
+    {
+      name: "reason",
+      type: "...string",
+      defaultValue: "No reason given",
+    },
+  ],
+  userServerPermissions: [
+    "KICK_MEMBERS",
+  ],
+  botServerPermissions: [
+    "KICK_MEMBERS",
+  ],
+  execute: (message, _args, guild) => {
+    const member: Member = _args["member"];
+    const reason: string = _args["reason"];
+
+    if (!guild || !member || !reason) return;
```

Comment:
Once you fix the args accessing, you will not need to check any of these. All of these will exist